### ISSUE TITLE
sentry-cli receives "Additional properties are not allowed ('debug_id' was unexpected)" when uploading DIF

### ISSUE THREAD

**Issue by alinradut** [getsentry/sentry-cli#1056]
### Environment

How do you use Sentry?
self-hosted 9.0.0

Which SDK and version?
sentry-cli 1.70.0

### Steps to Reproduce

Attempt to upload debug information files to the sentry server using sentry-cli:

`sentry-cli --url $URL --auth-token $AUTH_TOKEN upload-dif --log-level trace $PATH --org $ORG --project $PROJECT`

### Expected Result

Successful 200 response

### Actual Result

```
> POST /api/0/projects/ORG/PROJECT/files/difs/assemble/ HTTP/1.1
> Host: sentry.HOST
> Accept: */*
> Connection: TE
> TE: gzip
> User-Agent: sentry-cli/1.70.0
> Authorization: Bearer 30953cc0***
> Content-Type: application/json
> Content-Length: 21476
< HTTP/1.1 400 BAD REQUEST
< Server: nginx/1.15.9
< Date: Wed, 20 Oct 2021 18:44:50 GMT
< Content-Type: application/json
< Content-Length: 78
< Connection: keep-alive
< X-XSS-Protection: 1; mode=block
< Content-Language: en
< X-Content-Type-Options: nosniff
< Vary: Accept-Language, Cookie
< Allow: POST, OPTIONS
< X-Frame-Options: deny
response status: 400
body: {"error": "Additional properties are not allowed ('debug_id' was unexpected)"}
error: API request failed
  caused by: sentry reported an error: Additional properties are not allowed ('debug_id' was unexpected) (http status: 400)
client close; no transport to shut down  (from sentry)
```

**Comment by flub** [comment]
> This issue has gone three weeks without activity. In another week, I will close it.
> 
> But! If you comment or otherwise update it, I will reset the clock, and if you label it `Status: Backlog` or `Status: In Progress`, I will leave it alone ... forever!

Liar, it was labelled as backlog and you still came by :(

**Comment by flub** [comment]
> > This issue has gone three weeks without activity. In another week, I will close it.
> > But! If you comment or otherwise update it, I will reset the clock, and if you label it `Status: Backlog` or `Status: In Progress`, I will leave it alone ... forever!
> 
> Liar, it was labelled as backlog and you still came by :(

oh wait, i just can't read github 🤦

**Comment by jan-auer** [comment]
@kamilogorek This is a field we should keep empty depending on what the capability check responds with.

**Comment by kamilogorek** [comment]
Should be fixed in `sentry-cli@1.72.1`.
